### ISSUE TITLE
BLOCKSERVER: Config not reloading after component block change

### ISSUE THREAD

**Issue by John-Holt-Tessella** [ISISComputingGroup/IBEX#4006]
On LOQ we did:

1. Define component with a block
1. Save and switch to config containing component
1. Right click on component and change where the block is pointing to
1. Block does not change where it is pointing

There is a bug somewhere that means the config is not being reloaded on block definition change.

Is this responsible for #3884 as well?
Happend around 13/02/2019 at 9:20. Then did g.reload_config then it worked and everything was fine.
Block to look at is Changer which was pointing at LKUP:SAMPLE now points at LKUP:DLS

I have the relevant logs in the isis private share under log_for_ticket_4406_block_server_not_restarting_gateway

**Comment by Alistair-McGann-Tessella** [comment]
I have added an extra log into the share today (`LOQ00106649.log`) which shows that the blockserver is not updating with the new PV. Block `Changer` reads incorrectly `DLS4` until around 16:30. At this point I did g.reload_current_config() and the PV updated. This is reflected in the sample environment log file as the block starts reading `KB`, the correct value.

I could not reproduce this bug on NDXDEMO, we may have to debug this on NDXLOQ.
